source/_integrations/immich.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: Immich
+description: Instructions on how to integrate an immich user account into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2025.6
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_domain: immich
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@mib1185'
+ha_integration_type: service
+---
+
+This integration allows adding an [immich](https://immich.app/) user account to Home Assistant.
+
+## Prerequisites
+
+You need to [obtain the API key](https://immich.app/docs/features/command-line-interface#obtain-the-api-key) for your user account in your immich instance.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+URL:
+  description: "The URL of your immich instance. (_e.g. `https://immich.example.com`_)."
+API key:
+  description: "API key of your user account to connect to your immich instance."
+Verify SSL certificate:
+  description: "Whether to verify the SSL certificate when SSL encryption is used to connect to your immich instance."
+{% endconfiguration_basic %}
+
+## Data fetching
+
+Data are polled every 60 seconds.
```

Comment:
```suggestion
Data is polled every 60 seconds.
```

Reply:
I've rephrased it meanwhile 3 or 4 times, everytime the code:rabbit2: did complain it in another way :see_no_evil: at the moment i suggests "The integration polls data every 60 seconds."

Reply:
Sounds awesome.

Reply:
```suggestion
The integration polls data every 60 seconds.
```